multi_import/import_diff_applier.py

```diff
@@ -54,49 +61,49 @@ def create_object(self, obj_data):
         instance.save()
         return instance
 
-    def set_one_to_many(self, instance, mapping, values):
-        object_manager = getattr(instance, mapping.field_name)
+    def set_one_to_many(self, instance, field, values):
+        object_manager = getattr(instance, field.source)
```

Comment:
I've added an `update_instance` method to fields/relations, and moved this code to `ManyRelatedField`.

Fields can override the default behaviour, and I've done this for our Tags field. Our Tags field now calls `add_tag` and `remove_tag` on the model instance.